client/components/me-alerts/me-alerts.controller.js

```diff
@@ -0,0 +1,24 @@
+class MeAlertsCtrl {
```

Comment:
but it was already in place (page + service) and we tend to name controller based on the associated page name.